_upcoming-technologies/02-ipp-scan.md

```diff
@@ -1,6 +1,5 @@
 ---
 title: IPP scan (or virtual MF device) server (Scanner Application)
 ---
-The Printer Working Group (PWG) has also taken multi-function devices into account, devices which offer printer and scanner in one unit. For that they also have added IPP Scan, a driverless scanning standard for the scanners in IPP network multi-function devices.
```

Comment:
I had removed the previous version to prevent repetition. As discussed, I have added "The Printer Working Group (PWG) has also taken multi-function devices into account, devices which offer printer and scanner in one unit" in the new version.